tests/test_mesos_tools.py

```diff
@@ -422,3 +432,103 @@ def test_get_paasta_execute_docker_healthcheck_when_not_found():
         side_effect=fake_container_info,
     )
     assert mesos_tools.get_container_id_for_mesos_id(mock_docker_client, fake_mesos_id) is None
+
+
+@mark.parametrize('test_case', [
+    [['taska', 'taskb'],  # test_case0 - OK
+     [['outlna1', 'outlna2', 'errlna1'],
+      ['outlnb1', 'errlnb1', 'errlnb2']],
+     ['taska', 'outlna1', 'outlna2', 'errlna1', 'taskb', 'outlnb1', 'errlnb1', 'errlnb2'],
+     False],
+    [['a'],  # test_case1 - can't zip different length lists
+     [1, 2],
+     None,
+     True]])
```

Comment:
This is kind of hard to read - can you split these test cases out into individual tests?